### ISSUE TITLE
SAM2's setting during training

### ISSUE THREAD

**Issue by user8818** [SitongGong/Veason-R1#1]
Hello, thank you for sharing this great work!

I have carefully read your preprint and I am very interested in the studied task. Currently, I am preparing to reproduce Veason-R1 on my own device.
I was wondering how did you configure the SAM2 segmentation model during training? Was it kept completely frozen, and will this influence the model's performance? I could not find detailed instructions about this aspect in the paper’s implementation details, so I would really appreciate it if you could clarify.

Thank you in advance for your time and guidance!

**Comment by hong-hd** [comment]
> 您好，感谢您分享这项伟大的作品！
> 
> 我仔细阅读了您的预印本，对所研究的任务非常感兴趣。目前，我正准备在自己的设备上复现 Veason-R1。 我想知道您在训练过程中是如何配置 SAM2 分割模型的？它是否保持完全冻结状态？这会影响模型的性能吗？我在论文的实现细节中找不到关于这方面的详细说明，所以如果您能解释一下，我将不胜感激。
> 
> 提前感谢您的时间和指导！

Hi, have you successfully reproduced the code?

**Comment by hong-hd** [comment]
> Hello, thank you for sharing this great work!
> 
> I have carefully read your preprint and I am very interested in the studied task. Currently, I am preparing to reproduce Veason-R1 on my own device. I was wondering how did you configure the SAM2 segmentation model during training? Was it kept completely frozen, and will this influence the model's performance? I could not find detailed instructions about this aspect in the paper’s implementation details, so I would really appreciate it if you could clarify.
> 
> Thank you in advance for your time and guidance!

Hi, have you successfully reproduced the code?

**Comment by user8818** [comment]
来信已收到。将尽快回复。

**Comment by SitongGong** [comment]
Hi, thank you for your attention to our work, and we apologize for the delay. 
For your question, we kept the SAM2 frozen for mask propagation during all the training stages. We now release all the training code in this repo. Please feel free to reference its implementation details.

**Comment by user8818** [comment]
来信已收到。将尽快回复。
